javascript/javascript1/week4/voiceassistant.js

```diff
@@ -0,0 +1,81 @@
+let toDo = ["gym", "football", "coding", "reading"];
+let myName;
+const helloCommand = "hello my name is";
+let num1;
+let num2;
+
+function getReply(command) {
+  if (command.indexOf(helloCommand) === 0) {
+    myName = command.slice(helloCommand.length + 1);
+    return "nice to meet you " + myName;
+  } else if (command === "what is your name") {
+    return "your name is " + myName;
+  } else if (command === "add fishing to my to do") {
+    toDo.push("fishing");
+    console.log(toDo);
+  } else if (command === "remove fishing from my to do") {
+    const toRemoveToDo = toDo.indexOf("fishing");
+    if (toRemoveToDo > -1) {
+      toDo.splice(toRemoveToDo, 4);
+    }
+    console.log(toDo);
+    return "Removed fishing from your todo";
+  } else if (command === "What is on my todo?") {
+    let toDoLength = toDo.length;
+    return;
+    "you have " + toDoLength + " todos - gym,football,coding and reading";
```

Comment:
You should return the string, line number 26 is Unreachable this way because it is after return.
You can also return the array itself 
```js
return "you have " + toDo.length + " todos -" + toDo;
```